projects/Amlogic/bootloader/update.sh

```diff
@@ -75,12 +75,14 @@ for arg in $(cat /proc/cmdline); do
         esac
       fi
 
-      for all_dtb in /flash/*.dtb /flash/DTB; do
-        dtb=$(basename $all_dtb)
-        if [ -f $SYSTEM_ROOT/usr/share/bootloader/$dtb ]; then
-          echo "*** updating Device Tree Blob: $dtb ..."
-          mount -o rw,remount $BOOT_ROOT
-          cp -p $SYSTEM_ROOT/usr/share/bootloader/$dtb $BOOT_ROOT
+      for all_dtb in /flash/*.dtb ; do
+        if [ -f $all_dtb ] ; then
+          dtb=$(basename $all_dtb)
+          if [ -f $SYSTEM_ROOT/usr/share/bootloader/$dtb ]; then
+            echo "*** updating Device Tree Blob: $dtb ..."
```

Comment:
Please use sentence casing so `Updating device tree blob:` and avoid `***` prefixes with on-screen messages as we don't includes them with the kernel and system messages. There are a couple of other `***` instances that need to be removed too.

Reply:
The patch is simply a bugfix, the only added thing is `if`: https://github.com/LibreELEC/LibreELEC.tv/pull/2502/files?w=1
"*** updating Device Tree Blob: $dtb ..." follows what was present in official LE 8.2: 
https://github.com/LibreELEC/LibreELEC.tv/blob/libreelec-8.2/packages/tools/u-boot/scripts/update-c2.sh#L38-L45
https://github.com/LibreELEC/LibreELEC.tv/blob/libreelec-8.2/packages/tools/u-boot/scripts/update.sh#L42-L50
If this requires a cleanup, I suggest it's done in a separate PR.